templates/download/desktop/index.html

```diff
@@ -128,9 +128,140 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
         </div>
         <!-- tab 2 -->
         <div class="p-tabs__content"
-             id="system-requirements"
+             id="system-requirements-{{ releases.lts.slug }}"
              role="tabpanel"
-             aria-labelledby="system-requirements-tab">
+             aria-labelledby="system-requirements-tab-{{ releases.lts.slug }}">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">2 GHz dual-core processor or better</li>
+            <li class="p-list__item is-ticked">4 GB system memory</li>
+            <li class="p-list__item is-ticked">25 GB of free hard drive space</li>
+            <li class="p-list__item is-ticked">Either a USB port or a DVD drive for the installer media</li>
+            <li class="p-list__item is-ticked">Internet access is helpful</li>
+          </ul>
+        </div>
+        <!-- tab 3 -->
+        <div class="p-tabs__content"
+             id="how-to-install-{{ releases.lts.slug }}"
+             role="tabpanel"
+             aria-labelledby="how-to-install-tab-{{ releases.lts.slug }}">
+          <p>To install or try Ubuntu Desktop:</p>
+          <ol class="p-list--divided">
+            <li class="p-list__item">Download the ISO image</li>
+            <li class="p-list__item">
+              Create a bootable USB flash drive with <a href="https://etcher.balena.io/">balenaEtcher</a> or similar
+            </li>
+            <li class="p-list__item">Boot your laptop or PC from the USB flash drive</li>
+          </ol>
+          <hr class="p-rule" />
+          <p>
+            <a href="/tutorials/install-ubuntu-desktop">Follow the step-by-step tutorial&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <div class="p-section">
+          <h2>Ubuntu {{ releases.latest.full_version }} LTS</h2>
+        </div>
+        <div class="p-image-wrapper u-hide--small u-hide--medium">
+          {{ image(url="https://assets.ubuntu.com/v1/3b5fa561-mascot-numbat@2x.png",
```

Comment:
We need to update to the new mascot asset, left a comment on the doc for that as well